### PR TITLE
Small adjustments to simplify tests readability

### PR DESCRIPTION
**Pull request by z4r4tu5tr4**
I tried to simplify the unit tests to the maximum while reading them, but I could not simplify everything and not insert all the tests in PEP8. I'll try another commit.

### CODE REVIEW

File: tests/test_processor.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import socket
-import unittest
+from socket import socket, AF_INET, SOCK_STREAM
```
Comment (kashishm) [inline tests/test_processor.py]: I prefer `import <module>` than `from <module> import <function>`. In my opinion using regular import improves readability. When reviewing Python code, I like seeing where the given function or class comes from right where it is used. It saves me from scrolling to the top of the module to get that info.

File: tests/test_processor.py [hunk 2]
```diff
@@ -31,8 +34,11 @@ def test_Processor_suite_data_store_init_request(self):
         processors[Message.SuiteDataStoreInit](None, response, None)
 
         self.assertEqual(Message.ExecutionStatusResponse, response.messageType)
-        self.assertEqual(False, response.executionStatusResponse.executionResult.failed)
-        self.assertEqual(0, response.executionStatusResponse.executionResult.executionTime)
+        self.assertEqual(False,
```
Comment (kashishm) [inline tests/test_processor.py]: I am not sure if this line is required to be split. I use PyCharm's default formatting and it doesn't complain for lines having less than 120 chars. I think we can follow that convention. But I'd be happy to hear your thoughts on this.

**Comment by z4r4tu5tr4** [comment]
I really agree with the choice to leave the modules always explicit, but I think in some cases, in the unittest classes for example, it's kind of obvious. But in the socket example, I think it can complicate readability with the large amount of 'socket.anything'. But his argument in this case is entirely valid.

However, I do not agree with the notation adopted by pycharm because the standard defined in PEP8, which is generally adopted by large python projects. For example, I'm using Atom with pylama plugin and my linter is really noisy when the code does not follow the PEP8 standard.

In short, I do not see many problems using 120 characters, but I think that to share with the community we should follow the good practices of python code. Avoiding errors like E501 and adapting the code to not have errors like E122.

I think if we try to use pylint and radon as criteria, we would have good readability and standardization results.

I'm sorry for my English, I do not have fluency and it may be that in the translation something sounded rude.

**Comment by kashishm** [comment]
@z4r4tu5tr4 Thanks for the PR! Really appreciate your contribution.
